packages/utils/src/index.ts

```diff
@@ -9,20 +9,21 @@ export {
   objectToCliArgs,
 } from './lib/execute-process';
 export { git, latestHash } from './lib/git';
-export { importModule } from './lib/load-file';
+export { importEsmModule } from './lib/load-file';
+export { FOOTER_PREFIX, CODE_PUSHUP_DOMAIN } from './lib/report';
 export { reportToMd } from './lib/report-to-md';
 export { reportToStdout } from './lib/report-to-stdout';
 export {
-  calcDuration,
-  compareIssueSeverity,
   countOccurrences,
   distinct,
-  formatBytes,
-  formatCount,
   objectToEntries,
   pluralize,
   readJsonFile,
   readTextFile,
-  slugify,
   toArray,
 } from './lib/utils';
+export { formatBytes } from './lib/report';
+export { slugify } from './lib/report';
+export { calcDuration } from './lib/report';
+export { formatCount } from './lib/report';
+export { compareIssueSeverity } from './lib/report';
```

Comment:
All the exports from `./lib/report` can be merged.

(VSCode optimizes imports/exports for me automatically on save, BTW. Maybe WebStorm also has some equivalent?)

![image](https://github.com/flowup/quality-metrics-cli/assets/34691111/209dd8dc-4eed-4eb4-813f-57f0f54c6817)

Reply:
done!